src/main/java/uk/gov/hmcts/cmc/claimstore/courtfinder/models/Address.java

```diff
@@ -15,4 +19,12 @@
     private List<String> addressLines;
     private String postcode;
     private String town;
+
+    @Override
+    public String toString() {
+        return Stream.of(addressLines, Arrays.asList(postcode, town))
+            .flatMap(Collection::stream)
+            .map(s -> s.replaceAll("[\\r\\n]", ""))
```

Comment:
This can be made more platform independent though.

Reply:
👍